javascript/javascript1/week3/javascript-warmup/Series-duration-of-my-life.js

```diff
@@ -0,0 +1,57 @@
+const seriesDurations = [
+    {
+      title: "Game of thrones",
+      days: 3,
+      hours: 1,
+      minutes: 0,
+    },
+    {
+      title: "Sopranos",
+      days: 3,
+      hours: 14,
+      minutes: 0,
+    },
+    {
+      title: "The Wire",
+      days: 2,
+      hours: 12,
+      minutes: 0,
+    },
+  ];
+  
+  function logOutSeriesText() {
+    // write code here
+    const eightyYearsIntoHours = 80 * 365 * 24;
+
+
+
+    // gameOfThrones Variables
+    const gameOfThronesdaysIntoHours = seriesDurations[0].days*24;
```

Comment:
This works as intended, but it has a lot of duplicated code, and what happens if a new series is added? 🔍 

I would like you to resolve these issues by implementing it with a loop instead. 😄